doc/release-notes-27679.md

```diff
@@ -1,2 +0,0 @@
-- unix socket paths are now accepted for `-zmqpubrawblock` and `-zmqpubrawtx` with
```

Comment:
nit: Adjusted this in the [Wiki Page](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/28.0-Release-Notes-Draft).  Line wrap issue.